### ISSUE TITLE
relayer's status is always "starting..."  in `roller run`

### ISSUE THREAD

**Issue by mtsitrin** [dymensionxyz/roller#140]
runnning `roller run` it shows
sequencer active
da active
relayer starting...

the relayer process is not running (validated with `psz`)
it probably fails due to #133 

the `roller run` command should change the relayer status to Failed

Edit:
Same error on the otherside as well
The relayer is up and running, 
The output still shows `Starting...`

**Comment by omritoptix** [comment]
As discussed:

1. Make sure update-client loop is always running
2. separate to 2 statuses: relaying... and creating connection...
3. make sure rly tx relay-packets is always running after connection creation

**Comment by mtsitrin** [comment]
blocked by #179

**Comment by mtsitrin** [comment]
As discussed, when the status is active it should reflect the Channels used

**Comment by ItayLevyOfficial** [comment]
closed as duplicate of #190
